StitchCoreServicesMongoDbRemote-iOS/StitchCoreServicesMongoDbRemote-iOS/RemoteMongoCollection.swift

```diff
@@ -0,0 +1,249 @@
+import Foundation
+import MongoSwift
+import StitchCore_iOS
+import StitchCoreServicesMongoDbRemote
+
+/**
+ * A class representing a MongoDB collection accesible via the Stitch MongoDB service. Operations against the Stitch
+ * server are performed asynchronously.
+ */
+public class RemoteMongoCollection<T: Codable> {
```

Comment:
[nit][minor] All functions should have labels.

Reply:
The lack of labels is following suit with swift driver. I could go either way on having them or not. Not having makes it faster to write in arguments at the cost of not knowing the argument name written in.

Reply:
Keeping as is per SDK chat discussion.